File: src/models/RBS/Department.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Rch.Driver.Net.Models.RBS
{
    public class Department
    {
        public int vat{ get; set; }
        public int halo{ get; set; }
        public int lalo{ get; set; }
        public string name{ get; set; }
        public int index{ get; set; }
        public int price{ get; set; }
        public bool single{ get; set; }
        public string dpt_type{ get; set; }
        public bool pay_discount { get; set; }
    }
}

File: RCH Driver JS/src/interfaces/department.ts
/**
 * @author: Luca Fabbri <l.fabbri@rch.it>
 * @exports IDepartment
 */
export interface IDepartment {
  id: number;
  name: string;
  departmentType: number;
  price: number;
  halo: number;
  lalo: number;
  single: boolean;
  vatCode: number;
  groupCode: number;
}
